internal/weldr/upload.go

```diff
@@ -99,32 +118,26 @@ func TargetsToUploadResponses(targets []*target.Target) []UploadResponse {
 	return uploads
 }
 
-func UploadRequestToTarget(u UploadRequest, filename string) (*target.Target, error) {
```

Comment:
Hm, you added the parameter in the previous commit, just to remove it again here. It is never used.

Reply:
Ah, a rebase gone awry. Thanks! Fixed and pushed two natural follow-ups on top.